core/templates.yml

```diff
@@ -374,6 +374,15 @@ paths:
     sequence_asset_publish_texture:         '@sequence_asset_publish_area_texture/@texture_subdirs/@sequence_asset_version_name[.{UDIM}].{tex.ext}'
     shot_asset_publish_texture:             '@shot_asset_publish_area_texture/@texture_subdirs/@shot_asset_version_name[.{UDIM}].{tex.ext}'
 
+    # MIPMAPS
+    project_publish_mipmap:                 '@project_publish_area_texture/@texture_subdirs/mipmaps/@project_version_name[.{UDIM}].{tex.ext}'
+    sequence_publish_mipmap:                '@sequence_publish_area_texture/@texture_subdirs/mipmaps/@sequence_version_name[.{UDIM}].{tex.ext}'
+    shot_publish_mipmap:                    '@shot_publish_area_texture/@texture_subdirs/mipmaps/@shot_version_name[.{UDIM}].{tex.ext}'
+
+    project_asset_publish_mipmap:           '@project_asset_publish_area_texture/@texture_subdirs/mipmaps/@project_asset_version_name[.{UDIM}].{tex.ext}'
+    sequence_asset_publish_mipmap:          '@sequence_asset_publish_area_texture/@texture_subdirs/mipmaps/@sequence_asset_version_name[.{UDIM}].{tex.ext}'
+    shot_asset_publish_mipmap:              '@shot_asset_publish_area_texture/@texture_subdirs/mipmaps/@shot_asset_version_name[.{UDIM}].{tex.ext}'
+
```

Comment:
I believe the DDNA standard is to name the subdir after the file extension, so instead of creating separate templates for the mipmaps, we should probably just update `texture_subdirs` to include `{tex.ext}` at the end.

Reply:
I talked to Deke about this, the `mipmaps` subdir is different than what DDNA does, but I think its fine to move forward with that instead of the `{tex.ext}` subdir. I don't think we need both.

Reply:
I guess this is resolved then.